plugin.cfg

```diff
@@ -44,6 +44,7 @@ hosts:hosts
 route53:route53
 federation:federation
 kubernetes:kubernetes
+extkubernetes:extkubernetes
```

Comment:
don't really like the name, should we just go for external, or externelkubernetes or ... something?